e2e/smoke-tests-suite/panels_smokescreen.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { e2e } from '@grafana/e2e';
+
+// TODO: Better way to generate this list (ideally including all panels >=beta state) (list is case sensitive)
+const PANELS_TO_CHECK = [
+  'Time series',
+  'Bar chart',
+  'Stat',
+  'Gauge',
+  'Bar gauge',
+  'Table',
+  'Pie chart',
+  'State timeline',
+  'Heatmap',
+  'Status history',
+  'Histogram',
+  'Text',
+  'Alert list',
+  'Dashboard list',
+  'News',
+  'Annotations list',
+  'Candlestick',
+  'Canvas',
+  'Flame Graph',
+  'Geomap',
+  'Logs',
+  'Node Graph',
+  'Traces',
+  'XY Chart',
+];
+
+e2e.scenario({
+  describeName: 'Panels smokescreen',
+  itName: 'Tests each panel type in the panel edit view to ensure no crash',
+  addScenarioDataSource: false,
+  addScenarioDashBoard: false,
+  skipScenario: false,
+  scenario: () => {
+    e2e.flows.addDashboard();
+
+    // TODO: Try and use e2e.flows.addPanel() instead of block below
```

Comment:
Issue I ran into was on this line of internal method `configurePanel`: https://github.com/grafana/grafana/blob/3195d5bb0ad11ca2c33ea23822d538c990bdfe01/packages/grafana-e2e/src/flows/configurePanel.ts#L161